services/api/db/renderer.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import renderers
+import json
+
+class UserRenderer(renderers.JSONRenderer):
```

Comment:
the following screen shot is from the edison-api settings
![image](https://github.com/kalvilabs/kalvios/assets/142379213/93d371f9-7c72-4a04-bdb2-19588883029f)

If the defualt renderer class be defined like the above, will be still be needing this UserRenderer class defined in this file? Please share your thoughts?

Reply:
By creating a separate renderer class file, you promote modularity in your codebase. You can reuse the renderer class across multiple views or even across different Django apps within your project. This makes your code more maintainable and easier to understand. I knew this one line line setting in settings.py but we may need separate rendering classes for different view classes . It's better practice since we don't have large codebase yet so renderer class have no modification. Having said so if we feel like we don't need additional renderer class , we'll remove that. .